docs/topics/runbooks/dev/shell_into_dev_pod.md

```diff
@@ -0,0 +1,59 @@
+# Shell into a dev pod
```

Comment:
This is duplicating SRE's instructions in https://mana.mozilla.org/wiki/display/SRE/AMO+Dev+Resources, no?  I don't think we want to have it in two places, and (hopefully) SRE would maintain theirs if they change something.

I'm sure they wouldn't mind if we want to improve their docs though.

Reply:
I'd like to have all of our runbooks in one place. We can coordinate with SRE but there's no reason this info couldn't live here instead of on confluence.

Reply:
We could also just link there from here.